apps/docs/components/JwtGenerator/JwtGenerator.tsx

```diff
@@ -24,8 +24,41 @@ const serviceToken = `
 }
 `.trim()
 
+const generateRandomString = (length: number) => {
+  const CHARS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'
+  let result = ''
+
+  /**
+   * The number of possible Uint8 integers is 256. Since the length of `CHARS`
+   * doesn't fit exactly into 256, simply taking the modulus would create an
+   * uneven distribution that favors the earlier characters. To make a truly
+   * uniform distribution, we have to discard everything above the last full
+   * cycle, and pick again.
+   *
+   * The minus 1 is to account for 0-indexing.
+   */
+  const MAX = Math.floor(256 / CHARS.length) * CHARS.length - 1
+
+  const randomUInt8Array = new Uint8Array(1)
+
+  for (let i = 0; i < length; i++) {
```

Comment:
@charislam did you mean this instead?
```suggestion
  for (let i = 0; i < CHARS.length; i++) {
```

Reply:
@kangmingtay, no I'm pretty sure this should be `length` from the arguments, which controls the length of the generated string. Each run through the loop generates one character for the random string

Reply:
oops missed that, ok lgtm then!